server/models/handlers/availability.js

```diff
@@ -1,6 +1,18 @@
 // Gets the time availabilities for a specific worker for a specific school.
+const _ = require('lodash');
+const workerModel = require('../db/worker');
+const UserTypes = require('../../constants/userTypes.json');
+
 async function getWorkerAvailability(workerId, schoolId, startTime, endTime) {
-    const sampleAvailableTimes = [
+   
+    // Searches the worker table to see if it is valid worker credentials
+    const worker = await workerModel.getWorker(email, password);
+    if  (!_.isNil(worker)) return worker;
+
+    // Returns null if the credentials do not match a student nor a worker's credentials.
+    return null;
+
+    const availableTimes = [
```

Comment:
Hi Nancy, 
This ticket involves implementing the logic of fetching the available times for a service worker. Thus, I would recommend making a database query to fetch the available times for the worker rather than returning the hardcoded values.